Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -464,7 +556,9 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
         
         if ($($ServiceFabricSkipped | Measure-Object).Count -gt 0)
         {
+
             Write-Host "`nError while setting up the cluster protection level in Service Fabric(s) in the subscription:" -ForegroundColor $([Constants]::MessageType.Error)
+            Write-Host $([Constants]::SingleDashLine)
```

Comment:
There will be also changes in the else block of -DryRun condition. Kindly look at app service PR for reference